KUBERNETES.md

```diff
@@ -0,0 +1,12 @@
+
+```bash
+
+kubectl create --filename _secrets/twitter-api-secret.yaml
```

Comment:
I always use long forms when documenting stuff or for snippets to copy/paste. It's easier to read and understand. `-f` could be "file" or "force".